gdk-pixbuf/sys/Cargo.toml

```diff
@@ -21,7 +21,7 @@ shell-words = "1.0.0"
 tempfile = "3"
 
 [features]
-dox = []
+dox = ["glib/dox", "gio/dox", "gobject/dox"]
```

Comment:
You will have to teach gir as it is what generates the sys/Cargo.toml to do this automatically.

Reply:
Okay, I'll try that instead.